src/main/webapp/script.js

```diff
@@ -682,6 +684,87 @@ function generateRainbowTags() {
   }
 }
 
+/**
+ * Sets value of a cookie.
+ *
+ * @param {string} cname name of the cookie
+ * @param {string} cvalue name of the value
+ */
+function setCookie(cname, cvalue) {
+  document.cookie = cname + '=' + cvalue + '; ';
+}
```

Comment:
Looks like setCookie only supports a single key-value pair for now while getCookie below supports multiple key-value pairs?

Reply:
Never mind. I misunderstood the Javascript cookies API.